test/Debugger/JsDiagGetFunctionPosition.js.dbg.baseline

```diff
@@ -31,8 +31,8 @@
   },
   {
     "functionPosition": {
-      "scriptId": 5,
-      "parentScriptId": 4,
+      "scriptId": 6,
+      "parentScriptId": 5,

```

Comment:
Are there any tools that rely on these indexes directly? Just food for thought that some updates might be needed.